content/src/content/jcr_root/apps/core/email/components/image/v1/image/readme.md

```diff
@@ -0,0 +1,55 @@
+<!--
+Copyright 2021 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Image Component
+
+Extensible image component for the Core Email Components for composing campaign content written in HTL and based on the [Image Core Component](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/image/v3/image)
+
+The Image Component allows embedding images in campaign content.
+
+## Features
+
+* Linkable to content pages, external URLs or page anchors
+* Allows an icon identifier to be configured for rendering an icon
```

Comment:
How does this work? In the UI, there is no indication of this feature. If it is considered "advanced" or only for developers, we need to document it more clearly here. If It is expected that the author user can use this feature, we will need to include it in the Experience League docs as well.

Reply:
removed